### ISSUE TITLE
[Bug-gmx_MMPBSA]: Issue creating mdin file

### ISSUE THREAD

**Issue by Tangbbmc** [Valdes-Tresanco-MS/gmx_MMPBSA#333]
### Bug summary

IndexError: string index out of range

### Terminal output

```bash
[INFO   ] Loading and checking parameter files for compatibility...
 File "/home/twg/miniconda3/envs/gmxMMPBSA/bin/gmx_MMPBSA", line 8, in <module>
    sys.exit(gmxmmpbsa())
  File "/home/twg/miniconda3/envs/gmxMMPBSA/lib/python3.9/site-packages/GMXMMPBSA/app.py", line 100, in gmxmmpbsa
    app.file_setup()
  File "/home/twg/miniconda3/envs/gmxMMPBSA/lib/python3.9/site-packages/GMXMMPBSA/main.py", line 127, in file_setup
    create_inputs(INPUT, self.normal_system, self.pre)
  File "/home/twg/miniconda3/envs/gmxMMPBSA/lib/python3.9/site-packages/GMXMMPBSA/createinput.py", line 213, in create_inputs
    gb_mdin = SanderGBInput(INPUT)
  File "/home/twg/miniconda3/envs/gmxMMPBSA/lib/python3.9/site-packages/GMXMMPBSA/createinput.py", line 264, in __init__
    self.mdin.change(self.parent_namelist[key], key, self.input_items[key])
  File "/home/twg/miniconda3/envs/gmxMMPBSA/lib/python3.9/site-packages/parmed/amber/mdin/mdin.py", line 197, in change
    if (value[0] == value[-1] == "'") or (value[0] == value[-1] == '"'):
IndexError: string index out of range
Exiting. All files have been retained.
```

### gmx_MMPBSA.log

[gmx_MMPBSA.log](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/files/10754153/gmx_MMPBSA.log)

### Operating system

Ubuntu 18.04

### gmx_MMPBSA Version

gmx_MMPBSA v1.5.7

### Python version

 PYTHON VERSION: 3.9.16 

### Installation

conda AmberTools + pip

**Comment by marioernestovaldes** [comment]
Please, install ParmEd from our reporsitory and and see how it goes:

`python -m pip install git+https://github.com/Valdes-Tresanco-MS/ParmEd.git@v3.4`

**Comment by Tangbbmc** [comment]
Ok. Thank you for your reply! I have solved this issue by using ParmEd 3.4.4.
